Makes the timeout for multiple keypress configurable (#12049)
nvaccess/nvda: user_docs/en/userGuide.md

This option allows the user to control if key presses generated by applications such as on-screen keyboards and speech recognition software should be processed by NVDA.
This option is on by default, though certain users may wish to turn this off, such as those typing Vietnamese with the UniKey typing software as it will  cause incorrect character input.

> ##### Maximum delay between two key presses for a repeated key press {#MaxRepeatedKeyDelay}

Author: The Rabbit suggests to improve this paragraph:
* Mention the default value and the range of acceptable values.
* Provide an example scenario where increasing the delay might be beneficial.
* Ensure consistency with the rest of the document by using a similar structure for settings 

The second point does not seem interesting to me: it just makes the documentation more verbose.
Though, the first and third point seem interesting since we are now documenting quite systematically the possible values and the default for each new option. For now we have only documented checkboxes / binary flags (Enabled/disabled) and combo-boxes / list of multiple choices.
Do we want to generalize this? If yes, how? Or is there a risk that the user becomes always more a technical documentation. E.g.:
Value range, value type (boolean, integer, list of strings), default value...

Cc @seanbudd, @LeonarddeR, @Adriani90 for opinion

Reviewer: I would not be too verbose here. I only agree with the rabbit's point number one. Mentioning the default value, and how it translates to seconds for the math-averse, seems like all we should do. I would not get overly formal, either.

> This option allows to control the maximum delay between two key presses to perform a double or triple key presses.

Author: Very nice. Thanks for the rephrasing and more detailed description.

> Increasing this delay may be especially useful for people using sticky keys or having some physical disability.

Reviewer: I think "delay" should be changed to "timeout", but that is only applicable if you accept my earlier idea. For now, this is just an English language clarification.

```suggestion
Increasing this delay may be especially useful for people using sticky keys, or who have a physical disability.
```

This option allows the user to control if key presses generated by applications such as on-screen keyboards and speech recognition software should be processed by NVDA.
This option is on by default, though certain users may wish to turn this off, such as those typing Vietnamese with the UniKey typing software as it will  cause incorrect character input.

> ##### Multiple press timeout {#MultiPressTimeout}

Reviewer 2: ```suggestion
##### Multiple key press timeout {#MultiPressTimeout}
```

Some NVDA keyboard gestures perform different actions based upon how many times the same key is pressed in rapid succession.

> An example of this is the Report current character of navigator object command, which reports the character if pressed once, a phonetic description of the character if pressed twice, and the numeric value of the character if pressed three times.

Reviewer 2: ```suggestion
An example of this is the "Report current character of navigator object" command.
This command reports the character if pressed once, a phonetic description of the character if pressed twice, and the numeric value of the character if pressed three times.
```

> This option configures the timeout after which an additional press of the same key will start a new gesture, rather than being taken as an additional press of the first one.

Reviewer 2: ```suggestion
This option configures the timeout after which an additional press of the same key will start a new gesture, rather than being taken as a subsequent press of the first one.
For the example command, a shorter timeout will cause two presses to the report the current character twice, rather than the phonetic description.
```

This option allows the user to control if key presses generated by applications such as on-screen keyboards and speech recognition software should be processed by NVDA.
This option is on by default, though certain users may wish to turn this off, such as those typing Vietnamese with the UniKey typing software as it will  cause incorrect character input.

##### Multiple press timeout {#MultiPressTimeout}

Some NVDA keyboard gestures perform different actions based upon how many times the same key is pressed in rapid succession.
An example of this is the "Report current character of navigator object" command.
This command reports the character if pressed once, a phonetic description of the character if pressed twice, and the numeric value of the character if pressed three times.
This option configures the timeout after which an additional press of the same key will start a new gesture, rather than being taken as a subsequent press of the first one.

> For the example command, a shorter timeout will cause two presses to the report the current character twice, rather than the phonetic description.

Author: @seanbudd, wonder if one is clearer (and is OK in English or not); feel free to accept or decline and merge as is.

```suggestion
For the example command, a too short timeout will cause two presses to the report the current character twice, rather than the phonetic description.
```

Reviewer 2: Fixing another small typo with it

```suggestion
For the example command, a too short timeout will cause two presses to report the current character twice, rather than the phonetic description.
```
